src/components/forms/sections/OverviewInfo.tsx

```diff
@@ -59,17 +59,17 @@ const OverviewInfo = ({
       <Container fluid>
         {!isWinterStorageFormMode && <OldBerthInfo application={application} />}
 
-        <Row>
-          <Col xs={8} md={10} className="vene-overview-info__header">
+        <div className="vene-overview-info__header">
```

Comment:
May i ask why ?

Reply:
This is not really a place for grid for various reasons. Header and edit link are not really column-like elements in the layout. Also we want border-bottom throughout the row and applying it on the Row element would have lead to applying other styling and spacing on grid Row which it's not really ment for.

Reply:
Maybe we can wrap the header element on full-width column to keep consistency with the grid markup and other Rows in the overview...